This used to only fail on aarch64 and s390x on our CI builders and our pp64le
package builder.  From tor 0.4.7.11 also on our x86 CI builder.
--- a/src/test/test_util.c
+++ b/src/test/test_util.c
@@ -4633,21 +4633,6 @@ test_util_glob(void *ptr)
   TEST("file1"PATH_SEPARATOR"*");
   EXPECT_EMPTY();
 
-#ifndef _WIN32
-  // test wildcard escaping
-  TEST("\\*");
-  EXPECT_EMPTY();
-
-  if (getuid() != 0) {
-    // test forbidden directory, if we're not root.
-    // (Root will be able to see this directory anyway.)
-    tor_asprintf(&pattern, "%s"PATH_SEPARATOR"*"PATH_SEPARATOR"*", dirname);
-    results = tor_glob(pattern);
-    tor_free(pattern);
-    tt_assert(!results);
-  }
-#endif /* !defined(_WIN32) */
-
 #undef TEST
 #undef EXPECT
 #undef EXPECT_EMPTY
